apps/institution-service/cypress/support/commands.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference types="cypress" />
+import "@testing-library/cypress/add-commands";
```

Comment:
We need the following:

1. github action that runs this test suite
2. At minimum a fake cypress test, or add a ping endpoint and ping that
3. package.json commands to open and run cypress

Reply:
I'll add 1 and 2. For 3, these commands already existed in the package file, so weren't necessary to add in this PR.